.github/workflows/documentation-commit.yml

```diff
@@ -1,44 +1,35 @@
-# This workflow will install Python dependencies, run tests and lint with a
-# single version of Python. For more information see:
-# https://help.github.com/actions/language-and-framework-guides\
-# /using-python-with-github-actions
+# On each push in 14.0 branch,
+# AND if the coverage file changed,
+# build documentation branch and commit the changes
+# so that the changes are visible on the website
+# https://oca.github.io/OpenUpgrade/
 
 name: Build and commit documentation
 
 on:
   push:
-    branches: ["14.0"]
+    paths: ["docsource/modules130-140.rst"]
 
 jobs:
-  documentation:
+  documentation-commit:
     runs-on: ubuntu-latest
-    env:
-      DB: "openupgrade"
-      DB_HOST: "localhost"
-      DB_PASSWORD: "odoo"
-      DB_PORT: 5432
-      DB_USERNAME: "odoo"
-      DOWNLOADS: https://github.com/OCA/OpenUpgrade/releases/download/databases
-      ODOO: "./odoo/odoo-bin"
-      PGHOST: "localhost"
-      PGPASSWORD: "odoo"
-      PGUSER: "odoo"
     steps:
+      - name: Check out OpenUpgrade Documentation
+        uses: actions/checkout@v2
+        with:
+          ref: "documentation"
+
       - name: Set up Python
-        uses: actions/setup-python@v2
+        uses: actions/setup-python@v4
         with:
-          python-version: 3.7
+          python-version: '3.10'
       - name: Check out Odoo
         uses: actions/checkout@v2
         with:
           repository: odoo/odoo
-          ref: "14.0"
+          ref: "16.0"
```

Comment:
🤔

Reply:
Great minds think alike ;-) See : https://github.com/OCA/OpenUpgrade/pull/3859#discussion_r1186923334

Answered in the other PR.

In the meantime, I removed the commit. Can be merged in my opinion.

Reply:
Yes, I saw later, that's why I resolved the comment.